### ISSUE TITLE
Print more helpful error when file does not exist for coverage check

### ISSUE THREAD

**Issue by cooperlees** [facebookincubator/ptr#51]
## How is `ptr` not doing what you expect?
If you add a file into `require_coverage` in setup.(cfg|py) you get a KeyError stack trace, which is not as helpful as it could be.
```
Traceback (most recent call last):
  File "/tmp/test_aioexa/bin/ptr", line 10, in <module>
    sys.exit(main())
  File "/private/tmp/test_aioexa/lib/python3.7/site-packages/ptr.py", line 1059, in main
    args.venv_timeout,
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/asyncio/base_events.py", line 584, in run_until_complete
    return future.result()
  File "/private/tmp/test_aioexa/lib/python3.7/site-packages/ptr.py", line 972, in async_main
    venv_timeout,
  File "/private/tmp/test_aioexa/lib/python3.7/site-packages/ptr.py", line 915, in run_tests
    await asyncio.gather(*consumers)
  File "/private/tmp/test_aioexa/lib/python3.7/site-packages/ptr.py", line 674, in _test_runner
    force_black,
  File "/private/tmp/test_aioexa/lib/python3.7/site-packages/ptr.py", line 625, in _test_steps_runner
    stats,
  File "/private/tmp/test_aioexa/lib/python3.7/site-packages/ptr.py", line 191, in _analyze_coverage
    if coverage_lines[afile].cover < cov_req:
KeyError: 'aioexabgp/client/pipes.py'
```

## What is your suggestion to make this better?
Catch the KeyError, check if file exists and LOG a friendly saying so.
- I feel we should also error + exit so that the error does not get ignored

## Code/Bug example?
See stack trace above

## How can a developer reproduce this?
Add a non existent file into require_coverage

**Comment by adhaamehab** [comment]
I'm currently working on this!

**Comment by cooperlees** [comment]
Thanks! Please put a PR up and add a unittest for this. If you get stuck feel free to put a PR up and I can comment on it :)

**Comment by cooperlees** [comment]
Fixed by #64
